actioncable/test/remote_connection_test.rb

```diff
@@ -0,0 +1,54 @@
+require 'test_helper'
+require 'stubs/test_server'
+require 'minitest/autorun'
+
+class ActionCable::RemoteConnections::RemoteConnection::BaseTest < ActionCable::TestCase
+  setup do
+    @server            = TestServer.new
+    @remote_connection = ActionCable::RemoteConnections::RemoteConnection.new(
+      @server,
+      var1: 1, var2: 2, var3: 3)
+  end
+
+  test '::new' do
+    assert_instance_of ActionCable::RemoteConnections::RemoteConnection, @remote_connection
+    assert_instance_of TestServer, @remote_connection.send(:server)
+    assert_equal @server, @remote_connection.instance_variable_get(:@server)
```

Comment:
This is testing the implementation details of `RemoteConnection`. We can remove this whole test and only keep the test that assert the exception is raised.